### ISSUE TITLE
Video Freezes When you go 10 sec later

### ISSUE THREAD

**Issue by KaanAlper** [cuynu/ytvanced#118]

**Comment by cyberk1ng** [comment]
Yeah, it seems like it freezes generally after you log-in to your Google account through YouTube Vanced.

**Comment by cuynu** [comment]
Change other google account

**Comment by cuynu** [comment]
> If you press 2 times to screen, it's passes 10 sec, and when you do it 3 times video is freezing until you close app and reopen

buffer ?

**Comment by cuynu** [comment]
You not fill into issue form, issue will switch to discussion.
